samples/outlook/35-notifications/add-getall-remove.yaml

```
@@ -0,0 +1,146 @@
+host: OUTLOOK
+api_set:
+    Mailbox: '1.3'
+script:
+    content: |
+        $("#addProgress").click(addProgress);
+        $("#addInformational").click(addInformational);
+        $("#addInformationalPersisted").click(addInformationalPersisted);
+        $("#addError").click(addError);
+        $("#getAll").click(getAll);
+        $("#replace").click(replace);
+        $("#remove").click(remove);
+          
+        function addProgress() {
+          var id = $("#nofiticationId").val();
+          var details =
+            {
+              type: "progressIndicator",
+              message: "Progress indicator with id = " + id
+            };
+          Office.context.mailbox.item.notificationMessages.addAsync(id, details, handleResult);
+        }
+
+        function addInformational() {
+          var id = $("#nofiticationId").val();
+          var details =
+            {
+              type: "informationalMessage",
+              message: "Non-persistent informational notification message with id = " + id,
+              icon: "icon1",
+              persistent: false
+            };
+          Office.context.mailbox.item.notificationMessages.addAsync(id, details, handleResult);
+        }
+
+        function addInformationalPersisted() {
+          var id = $("#nofiticationId").val();
+          var details =
+            {
+              type: "informationalMessage",
+              message: "Persistent informational notification message with id = " + id,
+              icon: "icon1",
+              persistent: true
+            };
+          Office.context.mailbox.item.notificationMessages.addAsync(id, details, handleResult);
+        }
+
+        function addError() {
+          var id = $("#nofiticationId").val();
+          var details =
+            {
+              type: "errorMessage",
+              message: "Error notification message with id = " + id
+            };
+          Office.context.mailbox.item.notificationMessages.addAsync(id, details, handleResult);
+        }
+
+        function getAll() {
+          Office.context.mailbox.item.notificationMessages.getAllAsync(handleResult);          
+        }
+
+        function replace() {
+          var id = $("#nofiticationId").val();
+          Office.context.mailbox.item.notificationMessages.replaceAsync(
+            id,
+            {
+              type: "informationalMessage",
+              message: "Notification message with id = " + id + " has been replaced with an informational message.",
+              icon: "icon2",
+              persistent: false
+            },
+            handleResult);
+        }
+
+        function remove() {
+          var id = $("#nofiticationId").val();
+          Office.context.mailbox.item.notificationMessages.removeAsync(id, handleResult);
+        }
+
+        function handleResult(result) {
+          console.log(`${JSON.stringify(result)}`);
```

Can this statement be simplified to something like the following?
```suggestion
          console.log(JSON.stringify(result));
```